opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -448,6 +459,107 @@ message HistogramDataPoint {
   uint32 flags = 10;
 }
 
+// ExponentialHistogramDataPoint is a single data point in a timeseries that describes the
+// time-varying values of a ExponentialHistogram of double values. A ExponentialHistogram contains
+// summary statistics for a population of values, it may optionally contain the
+// distribution of those values across a set of buckets.
+//
+message ExponentialHistogramDataPoint {
+  // The set of key/value pairs that uniquely identify the timeseries from
+  // where this point belongs. The list may be empty (may contain 0 elements).
+  repeated opentelemetry.proto.common.v1.KeyValue attributes = 1;
+
+  // StartTimeUnixNano is optional but strongly encouraged, see the
+  // the detiled comments above Metric.
+  //
+  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
+  // 1970.
+  fixed64 start_time_unix_nano = 2;
+
+  // TimeUnixNano is required, see the detailed comments above Metric.
+  //
+  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
+  // 1970.
+  fixed64 time_unix_nano = 3;
+
+  // count is the number of values in the population. Must be non-negative. This
+  // value must be equal to the sum of the "count" fields in buckets if a
```

Comment:
shall we explicitly say that "count" also includes "zero_count"?

Reply:
Yes. I had copied the comments for `count` and `sum` from the explicit histogram point, and on re-review I see there are a couple of extraneous parts. The explicit histogram point mentions "if provided" for the bucket counts, and we do permit explicit histograms to be lacking in buckets, in which case the histogram becomes a sum/count data point.

I've removed the "if provided" from this point--we may as well use the explicit histogram point if a sum/count data point is needed. There was also a mention of a "sum" field in the buckets themselves, which is a reference to something very old and no longer applicable.